src/Terminal/Terminal.ts

```diff
@@ -254,9 +254,9 @@ export class Terminal implements ITerminal {
     const expGain = calculateHackingExpGain(server, player);
     server.weaken(CONSTANTS.ServerWeakenAmount);
     this.print(
-      `'${server.hostname}' security level weakened to ${server.hackDifficulty}. Gained ${numeralWrapper.formatExp(
-        expGain,
-      )} hacking exp.`,
+      `'${server.hostname}' security level weakened to ${server.hackDifficulty.toFixed(
+        3,
+      )} and Gained ${numeralWrapper.formatExp(expGain)} hacking exp.`,
```

Comment:
```suggestion
    this.print(`'${server.hostname}' security level weakened to ${server.hackDifficulty.toFixed(3)} ` +
      `and Gained ${numeralWrapper.formatExp(expGain)} hacking exp.`,
```
I don't know why it was ever formatted the way it was, but perhaps something like this would be more liked all round?

Reply:
Umm, the comma and the + are personal preference to be honest, they do the same thing really.

Reply:
Thanks for the suggestion. But, I'm going to keep it how I committed it.